core/domain/exp_services.py

```diff
@@ -1682,15 +1681,14 @@ def get_scaled_average_rating(ratings):
         return 0
     average_rating = get_average_rating(ratings)
     z = 1.9599639715843482
-    x = python_utils.divide((average_rating - 1), 4)
+    x = ((average_rating - 1) // 4)
```

Comment:
Just to check -- isn't it wrong to use integer division here? (And also in some other cases in this PR, like the one just above this -- the multiplication by 1.0 seems to suggest that a float return value was expected.)

It is possible that the old code that used python_utils.divide was incorrect. If so, it might be necessary to investigate each case and use the appropriate division operator...

Reply:
> Just to check -- isn't it wrong to use integer division here? (And also in some other cases in this PR, like the one just above this -- the multiplication by 1.0 seems to suggest that a float return value was expected.)
> 
> It is possible that the old code that used python_utils.divide was incorrect. If so, it might be necessary to investigate each case and use the appropriate division operator...

Yes, I got it. That's why I am getting backend test errors.